R/oi_inclusive_mobility.R

```diff
@@ -0,0 +1,147 @@
+#' Function to recategorize OSM data based on the recent Inclusive Mobility (IM) guide [UK](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1044542/inclusive-mobility-a-guide-to-best-practice-on-access-to-pedestrian-and-transport-infrastructure.pdf)
```

Comment:
The first line should be a short title, e.g. 

```
#' Function to recategorize OSM data based on the Inclusive Mobility (IM)

```

Reply:
done!